pkg/rehearse/jobs.go

```diff
@@ -396,6 +396,64 @@ func AddRandomJobsForChangedTemplates(templates []config.ConfigMapSource, toBeRe
 	return rehearsals
 }
 
+func getPresubmitForRegistryStep(step config.RegistryStep, configs config.ByFilename, prConfigPresubmits map[string][]prowconfig.Presubmit) (string, *prowconfig.Presubmit) {
+	for _, ciopConfig := range configs {
+		tests := ciopConfig.Configuration.Tests
+		repoPresubmits := prConfigPresubmits[ciopConfig.Info.Repo]
+		for _, test := range tests {
+			if test.MultiStageTestConfiguration != nil {
+				jobPrefix := fmt.Sprintf("%s-ci-%s-%s-%s-", "pull", ciopConfig.Info.Org, ciopConfig.Info.Repo, ciopConfig.Info.Branch)
```

Comment:
Can we DRY this out with the logic that actually creates the names to ensure they stay in sync?